core/src/main/java/org/mule/runtime/core/internal/source/scheduler/DefaultSchedulerMessageSource.java

```diff
@@ -82,8 +86,11 @@ public DefaultSchedulerMessageSource(MuleContext muleContext, PeriodicScheduler
     this.muleContext = muleContext;
     this.scheduler = scheduler;
     this.disallowConcurrentExecution = disallowConcurrentExecution;
-    this.notificationHelper =
-        new NotificationHelper(muleContext.getNotificationManager(), ConnectorMessageNotification.class, false);
+
+    notificationFunctions = new LinkedList<>();
+    // Add message received notification
+    notificationFunctions
+        .add((event, component) -> createConnectorMessageNotification(this, event, component.getLocation(), MESSAGE_RECEIVED));
```

Comment:
isn't this the exact same notification that the FlowProcessMediator is already sending?

Reply:
Yes. Just added a test

Reply:
https://github.com/mulesoft/mule-integration-tests/pull/810